data.js

```diff
@@ -812,4 +815,418 @@ exports.data = [
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "[2019](https://www.startnext.com/12062020)",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Hamburg muss handeln",
+            link: "http://hamburg-muss-handeln.de/",
+            keywords: ["private-sector", "legislation", "local", "grassroots"],
+            label: "hamburg-muss-handeln",
+        },
+        description: {
+            content: "A coalition of 300+ companies and organizations from Hamburg to express support for the demands of Fridays for Future. Also provides advice for upcoming state elections.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Fossil Free",
+            link: "https://gofossilfree.org/",
+            keywords: ["grassroots", "legislation", "private-sector", "network", "ngo"],
+            label: "fossil-free",
+        },
+        description: {
+            content: "A network of local grassroots initiatives that individually petition governments and companies to go fossil free. Local groups are present in virtually all major city and are independent in how they pursue their goals and who they talk to.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "[2012](https://gofossilfree.org/about/)",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Scientists for Future",
+            link: "https://www.scientists4future.org/",
+            keywords: ["science", "grassroots", "education", "ngo"],
+            label: "scientists-for-future",
+        },
+        description: {
+            content: "An interdisciplinary group of scientists working for a sustainable future. They support Fridays for Future and established [Lectures for Future](https://www.scientists4future.org/infomaterial/vortragsreihen/) to push the climate crisis onto the agenda in universities.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+        established: {
+            content: "Probably [2019](https://www.scientists4future.org/about/faq/)",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Europe Beyond Coal",
+            link: "https://www.beyond-coal.eu",
+            keywords: ["coal", "grassroots", "education", "network", "ngo"],
+            label: "europe-beyond-coal",
+        },
+        description: {
+            content: "They pressure governments, investors and insurers to stop supporting extraction and burning of coal in Europe. They track all planned and running [coal plants in the EU](https://beyond-coal.eu/data/) with data on health impact and emissions"
```

Comment:
Something more general would be great like: 'This organization is trying to provide public with information regarding their power supplies. This put pressure on ...' 
This shows that their focus is mostly inforamtion and education.

Reply:
Yep, like. Changed it to `They raise awareness with decision makers and the public about the impact of current and planned coal-based power-supplies. This puts pressure governments, investors and insurers to stop supporting extraction and burning of coal in Europe` online (as this repo is discontinued)

Reply:
Like!